src/feature-viewer.js

```diff
@@ -790,10 +790,27 @@ function createFeature(sequence, div, options) {
                         .style("stroke-opacity", 1);
                 }
             },
-            rectangle: function (object, position) {
+            rectangle: function (object, position, onClick) {
                 //var rectShift = 20;
                 if (!object.height) object.height = 12;
                 if(typeof object.showDescriptionRect === 'undefined' || object.showDescriptionRect === null) object.showDescriptionRect = true;
+                if(typeof object.summaryView === 'undefined' || object.summaryView === null) object.summaryView = false;
+                if(typeof object.summaryViewProperties === 'undefined' || object.summaryViewProperties === null) object.summaryViewProperties = {};
+                if(typeof object.summaryViewProperties.buttonColor === 'undefined' || object.summaryViewProperties.buttonColor === null) object.summaryViewProperties.buttonColor = "#5789d4";
```

Comment:
=== null, isn't simply if(object.summaryViewProperties.buttonColor) ?

Reply:
This was corrected